src/PSMDATP/Public/Get-MDATPDevice.ps1

```diff
@@ -120,7 +121,7 @@
         # Connect with MDATP API
         [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
         $Body = @{
-            resource      = "https://api.securitycenter.windows.com"
+            resource      = $Uri
```

Comment:
Changing this to regional affects the OAUTH process resulting in a failure message although the script does continue

```
Invoke-RestMethod : {"error":"invalid_resource","error_description":"AADSTS500011: The resource principal named
https://api-uk.securitycenter.windows.com was not found in the tenant
```